test/core/end2end/grpc_core_end2end_test.bzl

```diff
@@ -73,7 +73,100 @@ _DEPS = [
 ]
 
-def grpc_core_end2end_test(name, shard_count = 1, enable_fuzzing = True, tags = [], flaky = False):
+_TESTS = [
+    "bad_ping",
+    "binary_metadata",
+    "call_creds",
+    "call_host_override",
+    "cancel_after_accept",
+    "cancel_after_client_done",
+    "cancel_after_invoke",
+    "cancel_after_round_trip",
+    "cancel_before_invoke",
+    "cancel_in_a_vacuum",
+    "cancel_with_status",
+    "channelz",
+    "client_streaming",
+    "compressed_payload",
+    "connectivity",
+    "default_host",
+    "disappearing_server",
+    "empty_batch",
+    "filter_causes_close",
+    "filter_init_fails",
+    "filtered_metadata",
+    "graceful_server_shutdown",
+    "grpc_authz",
+    "high_initial_seqno",
+    "hpack_size",
+    "http2_stats",
+    "invoke_large_request",
+    "keepalive_timeout",
+    "large_metadata",
+    "max_concurrent_streams",
+    "max_connection_age",
+    "max_connection_idle",
+    "max_message_length",
+    "negative_deadline",
+    "no_logging",
+    "no_op",
+    "payload",
+    "ping",
+    "ping_pong_streaming",
+    "proxy_auth",
+    "registered_call",
+    "request_with_flags",
+    "request_with_payload",
+    "resource_quota_server",
+    "retry",
+    "retry_cancel_after_first_attempt_starts",
+    "retry_cancel_during_delay",
+    "retry_cancel_with_multiple_send_batches",
+    "retry_cancellation",
+    "retry_disabled",
+    "retry_exceeds_buffer_size_in_delay",
+    "retry_exceeds_buffer_size_in_initial_batch",
+    "retry_exceeds_buffer_size_in_subsequent_batch",
+    "retry_lb_drop",
+    "retry_lb_fail",
+    "retry_non_retriable_status",
+    "retry_non_retriable_status_before_trailers",
+    "retry_per_attempt_recv_timeout",
+    "retry_per_attempt_recv_timeout_on_last_attempt",
+    "retry_recv_initial_metadata",
+    "retry_recv_message",
+    "retry_recv_message_replay",
+    "retry_recv_trailing_metadata_error",
+    "retry_send_initial_metadata_refs",
+    "retry_send_op_fails",
+    "retry_send_recv_batch",
+    "retry_server_pushback_delay",
+    "retry_server_pushback_disabled",
+    "retry_streaming",
+    "retry_streaming_after_commit",
+    "retry_streaming_succeeds_before_replay_finished",
+    "retry_throttled",
+    "retry_too_many_attempts",
+    "retry_transparent_goaway",
+    "retry_transparent_max_concurrent_streams",
+    "retry_transparent_not_sent_on_wire",
+    "retry_unref_before_finish",
+    "retry_unref_before_recv",
+    "server_finishes_request",
+    "server_streaming",
+    "shutdown_finishes_calls",
+    "shutdown_finishes_tags",
+    "simple_delayed_request",
+    "simple_metadata",
+    "simple_request",
+    "streaming_error_response",
+    "timeout_before_request_call",
+    "trailing_metadata",
+    "write_buffering",
+    "write_buffering_at_end",
+]
+
+def grpc_core_end2end_test_suite(name, config_src, deps = [], shard_count = 50, enable_fuzzing = True, tags = [], flaky = False):
     """Generate one core end2end test
 
     Args:
```

Comment:
Please document the `config_src` and `deps` parameters.

Reply:
done